src/provider/view/components/Header/Header.scss

```diff
@@ -0,0 +1,100 @@
+@import '../../styles/_variables.scss';
+
+.header {
+    width: $window-width;
+    color: var(--primary-text-color);
+    user-select: none;
+
+    .title {
+        height: 50px;
+        padding: 10px 20px;
+        background: var(--header-color);
+        display: flex;
+        flex-direction: row;
+        align-content: center;
+        justify-content: space-between;
+
+        span {
+            vertical-align: middle;
+            align-self: flex-start;
+            font-weight: 500;
+            font-size: 22px;
+        }
+
+        .count {
+            font-size: 18px;
+            margin-left: 7px;
+            color: var(--dark-text);
+        }
+    }
+
+    .strip {
+        $font-size: 12px;
+        position: relative;
+        background: var(--strip-background-color);
+        display: flex;
+        padding: 5px 20px;
+        align-content: flex-start;
+        flex-direction: row;
+        justify-content: space-between;
+
+        span {
+            vertical-align: middle;
+            align-self: center;
+            justify-self: center;
+            letter-spacing: 1px;
+        }
+
+        .options {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            vertical-align: middle;
+
+            li {
+                align-self: center;
+                display: inline-block;
+                color: var(--secondary-text-color);
+                margin-right: 5px;
+
+                &.sort-button {
+                    cursor: pointer;
+                    font-weight: 500;
+
+                    &:hover {
+                        color: var(--secondary-text-color-highlight);
+                    }
+                }
+
+                &.active {
+                    color: var(--primary-text-color);
+                    cursor: auto !important;
+
+                    &:hover {
+                        color: var(--primary-text-color-highlight);
+                    }
+                }
+
+                &.detail {
+                    text-transform: uppercase;
+                    font-size: $font-size;
+                    color: var(--accent-color);
```

Comment:
What exactly is the difference between a `$font-size` type variable and a `var(--accent-color)` type variable?

Is there a reason to mix-and-match like this?

Reply:
`$variable` is a compile time variable so its actual value is inserted at compile time. `--variable` is a runtime variable that can be changed with javascript. Not useful now but if/when we add theming this will allow us to load and modify values on the fly.